CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
+## DEV
+
+* Pass user's full name to `SitewitService.create_account()` and
+  `SitewitService.change_account_owner()` methods.
```

Comment:
I think it's more precise to say that some of our model's methods were updated to use user's full name.  What do you think?